index.html

```diff
@@ -1499,7 +1499,7 @@ <h3>Esther Jung
                         <img src="img/team/jaimie.jpeg" class="committee-photo jaimie">
                         <figcaption class="caption text-center">
                             <h3>Jaimie Lee
-                                <small class="major">Public Health, Psychology, and Public Policy</small>
+                                <small class="major">Public Health, Psychology, & Public Policy</small>
```

Comment:
Hm, I think there's a typo here (should just be a ",")

Reply:
Since Jaimie has 3 majors listed, the structure is as follows: "major, major, & major".

Reply:
Ah, I see.